data/workshops/2022.md

```diff
@@ -40,28 +118,11 @@ program_committee:
     affiliation: FAMAF, UNC and Manas.Tech, Argentina
 ---
 
-## Scope
+OCaml Workshop 2022 was co-located with ICFP 2022, in Ljubljana, Slovenia
```

Comment:
What does co-located mean in this instance, as only one location is mentioned?

Reply:
Replaced "was co-located with" by "took place during"